src/providers/export/azureCustomVision.json

```diff
@@ -15,7 +15,7 @@
                 "visited",
                 "tagged"
             ],
-            "default": "all",
+            "default": "visited",
```

Comment:
this doesn't seem to make any difference

Reply:
@mydiemho What do you mean?  This should make all the difference regarding how the form controls get initialized.